alembic/versions/614911daf883_add_seasonal_to_time_resolution_enum.py

```diff
@@ -0,0 +1,145 @@
+"""add 'seasonal' to time_resolution enum
+
+Revision ID: 614911daf883
+Revises: 
+Create Date: 2017-07-17 17:00:43.066818
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '614911daf883'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+connection = op.get_bind()
+dialect = connection.dialect.name
+
+
+# This is a *highly* customized migration script, barely related to the
+# autogenerated script at all. Two reasons:
+#
+# 1) There are a lot of differences between the (test, and presumably prod)
```

Comment:
I particularly distrust the constraint changes in the migration for the copy of the prod database.

All this indicates that:

1. The SQLAlchemy declaration is missing some elements -- at least indexes. 
2. Either my copy of the production database is significantly different than the actual production database, or the test database is significantly different (in schema).
2. We need a test migration based on the real production database to find out what's really true here.